### PR TITLE
Monitor crashes on clients 

### PR DESCRIPTION
**Pull request by vasilemario**
I've added dependencies for Crashlytics and documented the process for deployment

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -72,6 +75,8 @@ dependencies {
     implementation 'com.squareup.retrofit2:adapter-rxjava:2.1.0'
     implementation 'com.squareup.okhttp3:logging-interceptor:3.9.1'
 
+    implementation 'com.google.firebase:firebase-core:16.0.0'
```
Comment (tudormarze) [inline app/build.gradle]: Please update to latest (currently not building in Studio 3.3.1)

File: app/build.gradle [hunk 2]
```diff
@@ -72,6 +75,8 @@ dependencies {
...
+    implementation 'com.crashlytics.sdk.android:crashlytics:2.9.8'
```
Comment (tudormarze) [inline app/build.gradle]: Please update to latest

File: build.gradle [hunk 3]
```diff
@@ -4,11 +4,16 @@ buildscript {
...
+        maven {
+            url 'https://maven.fabric.io/public'
+        }
     }
     dependencies {
 
         classpath 'com.android.tools.build:gradle:3.2.1'
         classpath "io.realm:realm-gradle-plugin:5.7.0"
+        classpath 'com.google.gms:google-services:4.0.2'
+        classpath 'io.fabric.tools:gradle:1.26.1'
```
Comment (tudormarze) [inline build.gradle]: Please update to latest

File: build.gradle [hunk 4]
```diff
@@ -4,11 +4,16 @@ buildscript {
...
         google()
+        maven {
+            url 'https://maven.fabric.io/public'
+        }
     }
     dependencies {
 
         classpath 'com.android.tools.build:gradle:3.2.1'
         classpath "io.realm:realm-gradle-plugin:5.7.0"
+        classpath 'com.google.gms:google-services:4.0.2'
```
Comment (tudormarze) [inline build.gradle]: Please update to latest

**Comment by aniri** [comment]
@vasilemario could you please check the suggested changes? also, there are some conflicts that have to be resolved.

File: app/build.gradle (hunk 1 above)
Comment (vasilemario) [reply app/build.gradle]: In order to have Crashlytics we will need to have a google-services.json file

File: app/build.gradle (hunk 2 above)
Comment (vasilemario) [reply app/build.gradle]: I've updated to the latest version

**Comment by aniri** [comment]
changed updated in https://github.com/code4romania/monitorizare-vot-android/pull/137 . closing this pull request as commits from it are included in the new pull request :) thanks!

**Pull request closed without merging**